### PR TITLE
[IMP] Optimization of process to get stock_card values

### PR DESCRIPTION
**Pull request by josemoralesp**

### CODE REVIEW

**Comment by hbto** [comment]
Could create a PR for two outstanding customer instances in order to check that 
we are not breaking anything

**Review by hbto — APPROVED** [review]

**Review by hbto — CHANGES_REQUESTED** [review]
There are some changes to apply yet, I have seen some actual data that differs, does not match, in two different instances

**Review by hbto — APPROVED** [review]
I have check and the cause of the mismatching values is not in this PR, but in another PR the one in TyP, so, there was changed the code to fix the problem.

Go ahead with merging

File: stock_card/model/stock_card.py [hunk 1]
```diff
@@ -416,6 +417,10 @@ def _stock_card_move_history_get(self, product_id, locations_ids=None):
         query = '''
             SELECT distinct
                 sm.id AS move_id, sm.date, sm.product_id, prod.product_tmpl_id,
+                sm.origin_returned_move_id AS origin_returned_move_id,
+                sm.move_dest_id AS move_dest_id,
+                sm.move_dest_id AS move_dest_id,
```
Comment (hbto) [inline stock_card/model/stock_card.py]: Delete This line is written twice

**Comment by josemoralesp** [comment]
PR: 
TYP: https://github.com/Vauxoo/typ/pull/306
LODI: https://github.com/Vauxoo/lodigroup/pull/658

**Comment by hbto** [comment]
as both are ok I will endorse this PR
+1

**Review by JulioSerna — APPROVED** [review]

**Comment by JulioSerna** [comment]
👍

**Pull request merged**